src/Wrapper.js

```diff
@@ -264,6 +264,8 @@ class Wrapper extends React.PureComponent {
           {progress => (
             <React.Fragment>
               <MenuPanel
+                account={account}
+                apps={apps.filter(app => app.hasWebApp)}
```

Comment:
Oops, I think this must've been a merge conflict. I don't think we need to add `apps` here?